src/memmove.c

```diff
@@ -0,0 +1,107 @@
+#include "mem_functions.h"
+#include <arm_neon.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+[[gnu::always_inline]] inline uint8x16x2_t vld1q_u8_x2(unsigned char const* ptr) {
```

Comment:
would it be better to just update the toolchain and get these intrinsics?

Reply:
yep! xpack hasn't repackaged 14.2 yet, so once that's done, I'll edit this and we can move forward.